catalog/models.py

```diff
@@ -38,4 +39,201 @@ def __str__(self) -> str:
+        null=True,
+        blank=True
+    )
+
+    date = models.DateField(
+        _('Scene Date')
+    )
+
+    cloud_cover = models.FloatField(
+        _('Percentage of cloud cover'),
+        null=True,
+    )
+
+    url_tms = models.CharField(
+        _('Tile path'),
+        max_length=511
+    )
+
+    path = models.CharField(
+        _('Image repository'),
+        max_length=511,
+    )
+
+    download_link = models.CharField(
+        _('Image repository'),
+        max_length=511,
+    )
+
+    download_available = models.BooleanField(
+        _("Download allowed"),
+        default=False,
+
+    )
+
+    max_native_zoom = models.IntegerField(
+        _('Maximum zoom scale'),
+        default=15,
+
+    )
+
+    co_cr = models.BigIntegerField(
+        _('Regional Coordenation code'),
+    )
+
+    ds_cr = models.CharField(
+        _('Regional Coordenation name'),
+        max_length=255,
+    )
+
+    co_funai = models.IntegerField(
+        _('Funai code'),
+    )
+
+    no_ti = models.CharField(
+        _('Name of Indigenous Lands'),
+        max_length=255,
+    )
+
+    municipio = models.CharField(
+        _('County'),
+        max_length=255,
+        blank=True,
+        null=True
+    )
+
+    sg_uf = models.CharField(
+        _('State aconymn'),
+        max_length=255,
+        blank=True,
+        null=True
+    )
+
+    resolution = models.IntegerField(
+        _('Spatial resolution'),
+        null=True,
+        blank=True,
+    )
+    
+    nu_latitude = models.CharField(
+        _('Numero latitude'),
+        max_length=255,
+        blank=True,
+        null=True
+    )
+
+    nu_longitude = models.CharField(
+        _('Numero longitude'),
+        max_length=255,
+        blank=True,
+        null=True
+    )
+
+    # geom = models.PolygonField(
+    geom = models.GeometryField(
+        _('Geometry Field'),
+        srid=4674,
+        null=True,
+        blank=True,
+    )
+
+    class Meta:
+        """"Meta class for `catalog.Catalogs` abstract model."""
+        app_label = 'catalog'
+        verbose_name = 'Abastract Catalog'
+        verbose_name_plural = 'Abastract Catalogs'
+        abstract = True
+        ordering = ['-data']
+
+    def __str__(self):
+        """Returns `catalog.Catalogs` string data.
+
+        Returns:
+            str: model data image.
+        """
+        return str(self.image)
+
+
+class Landsat8Catalog (Catalogs):
+    """Model for Landsat scenes catalog."""
+
+    orbita = models.CharField(
+        _('Satellite Path'),
+        max_length=3,
+        null=True,
+        blank=True,
+    )
+
+    ponto = models.CharField(
+        _('Satellite Row'),
+        max_length=3,
+        null=True,
+        blank=True,
+    )
+
+    class Meta:
+        """"Meta class for `catalog.Landsat8Catalog` model."""
+        app_label = 'catalog'
+        verbose_name = 'Landsat8 Catalog'
+        verbose_name_plural = 'Landsat8 Catalogs'
+
+
+class Sentinel2Catalog (Catalogs):
```

Comment:
Remove whitespace between classname and (

Reply:
Removed.